src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -585,9 +585,10 @@ def __repr__(self):
             return "Endpoint({}, {}, {})".format(self.url, self.method,
                                                  self.action)
 
-    def __init__(self):
+    def __init__(self, *args, **kwargs):
         logger.info('Initializing controller: %s -> %s',
                     self.__class__.__name__, self._cp_path_)
+        super(BaseController, self).__init__(*args, **kwargs)
```

Comment:
Do not pass through any param to `object.__init__()` as it does not support any extra argument and it break in case this happens.